deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -126,6 +126,18 @@ After the benchmark has been run, the indices and documents of the source and ta
 ./catIndices.sh
 ```
 
+## Configuring SigV4 access
+This solution, when deployed, will attempt to configure needed IAM permissions on its applicable Migration services (Traffic Replayer and Migration Console) to allow SigV4 communication between the service and the AWS OpenSearch Service or OpenSearch Serverless target cluster. On the flip side, the target cluster must be configured to allow these resources as well, which the following subsections will discuss. 
```

Comment:
Too colloquial. May be difficult for some people to understand. Particularly for those who are not native English speakers.

Reply:
Was it `On the flip side` haha, I can remove it

Reply:
Have updated now